monetdb/bootstrap.sh

```diff
@@ -8,6 +8,7 @@ monetdbd start $MONETDB_STORAGE
 
 # Create the database instance
 monetdb create db
+monetdb set nclients=128 db
```

Comment:
Do we have a ticket to explore the implications of this?

Reply:
https://team-1617704806227.atlassian.net/browse/MIP-620